packages/cozy-harvest-lib/src/components/AccountModal.jsx

```diff
@@ -4,10 +4,12 @@ import PropTypes from 'prop-types'
 import React, { Component } from 'react'
 
 import { withClient } from 'cozy-client'
+import { Account } from 'cozy-doctypes'
```

Comment:
cozy-doctypes is deprecated, you should not use it for new code.

You should also check in the cozy-client's models https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/models/account.js#L94